src/main/plugin/iso19139.ca.HNAP/templates/vector-unilingual.xml

```diff
@@ -104,7 +120,7 @@
       <gmd:citation>
         <gmd:CI_Citation>
           <gmd:title >
-            <gco:CharacterString>English-only HNAP</gco:CharacterString>
+            <gco:CharacterString>English-only HNAP2</gco:CharacterString>
```

Comment:
Why is this called HNAP2?

Reply:
I was trying to verify that changes were being picked up.  The inflate-metadata.xsl was undoing changes that I was making.  I'll change it back in the future.